### DISCUSSION TITLE
ETA of next point release?

### DISCUSSION THREAD

**Discussion by 75th** [ampache/ampache#3848] [Q&A] [answered]
6.2.1 has a bug where, when you edit the genres of an album, those changes are not propagated to the songs, even if you check the "overwrite" checkbox. This is fixed on `develop`, but the current state of develop has other issues (missing CSS on the installer page at the very least).

Is there a plan for when the next stable point release incorporating that change will be published? I'm currently trying to move my whole family from iTunes to Ampache, but bulk editing metadata on the fly is going to be an important part of making Ampache work for us, at least at first.

**Comment by lachlan-00** [comment, marked as answer]
i've been at my new job for a month now and things are starting to calm down.

This weekend i've merged patch6 up to the latest develop and i'll start testing that. i just have one more api function to finish before i'm ready and then test/release process is pretty quick now that it's mostly automated.

If i can get a good block of time it could be this week.

**Reply by SuperFola** [reply]
Congrats on the new job!

It's impressive that you manage a work and a project this size alongside

**Reply by lachlan-00** [reply]
it helps when you have an easy job. i'll be setting up testing tomorrow

**Reply by lachlan-00** [reply]
sorry for double posting keep an eye on the pull here  https://github.com/ampache/ampache/pull/3856
